dpnp/backend/tests/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+find_package(GTest REQUIRED)
+
+find_package(Threads REQUIRED)

```

Comment:
add information messages about results of these finds (like MKL or PSTL)

Reply:
`find_package(Threads REQUIRED)` calls `FindThreads` module, that can easily print finding results, raise `FATAL_ERROR`. Additionally I have added message, if lib are not found.